frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/common.embedding.unit.spec.tsx

```diff
@@ -39,7 +39,7 @@ describe("[OSS] embedding settings", () => {
           screen.getByRole("link", { name: "upgrade to a paid plan" }),
         ).toHaveProperty(
           "href",
-          "https://www.metabase.com/upgrade?utm_media=embed-settings&utm_source=oss",
+          "https://www.metabase.com/upgrade?utm_source=product&utm_medium=upsell&utm_content=embed-settings&source_plan=oss",
```

Comment:
Upgrade URL change. Please see the issue [description how it has changed](https://github.com/metabase/metabase/pull/47116#issue-2480439695).